README.md

```diff
@@ -72,6 +73,7 @@ Here you can read the blog post about how it is deployed on Shuttle: [https://bl
     - pet name (e.g. `capital-mosquito.txt`)
     - alphanumeric string (e.g. `yB84D2Dv.txt`)
     - random suffix (e.g. `file.MRV5as.tar.gz`)
+    - supports overriding the filename
```

Comment:
This is not an outstanding feature so let's not mention it here. Someone with no idea about `random_url` might get confused when they read this.

Reply:
sure, I'll remove it

Reply:
done